### PR TITLE
Add Baldemar to admin group

### PR DESCRIPTION
**Pull request by relotnek**
### Summary
Adding FF representation in owners group with Baldemar

### Why do you need this?
1. Represents the IT Operations Managment Lead Baldemar (Baldy) for Filecoin Foundation, which is a significant stakeholder in the Filecoin ecosystem
2. Baldy is the primary contact for IT operations for the Filecoin Foundation and manages similar systems for Filecoin Foundation
3. Baldy is charged with responsibility over systems and infrastructure that Filecoin Foundation supports including billing, monitoring, and user management
4. This account is individually assigned to Baldy and is not used for day-to-day GitHub activities

### CODE REVIEW

File: github/filecoin-project.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@ members:
     # 2. Is experienced adminstering other similar orgs (e.g., ipfs)
     # 3. Is an independent contractor (similar to @anorth)
     - BigLep
+    # Why @filecoinfoundation-inf?
+    # 1. Represents the IT Operations Managment Lead Baldemar (Baldy) for Filecoin Foundation, which is a significant stakeholder in the Filecoin ecosystem
+    # 2. Baldy is the primary contact for IT operations for the Filecoin Foundation and manages similar systems for Filecoin Foundation
+    # 3. Baldy is charged with responsibility over systems and infrastructure that Filecoin Foundation supports including billing, monitoring, and user management
+    # 4. This account is individually assigned to Baldy and is not used for day-to-day GitHub activities
```
Comment (BigLep) [inline github/filecoin-project.yml:30]: The main concern I see here is if this is an account where the credentials are stored in some shared 1password vault.  I wouldn't want someone other than Baldy to have access.

**Review by BigLep — APPROVED** [review]
The request and justification make sense to me.  Let's leave open for a day in case there are any other comments/concerns.

File: github/filecoin-project.yml (hunk 1 above)
Comment (relotnek) [reply github/filecoin-project.yml:30]: it's an individual account and not a shared account in this case

File: github/filecoin-project.yml [hunk 2]
```diff
@@ -3457,8 +3463,8 @@ repositories:
     has_discussions: false
     merge_commit_message: PR_TITLE
     merge_commit_title: MERGE_MESSAGE
-    secret_scanning_push_protection: false
-    secret_scanning: false
+    secret_scanning_push_protection: true
```
Comment (jennijuju) [inline github/filecoin-project.yml:3466]: what does this flag do and why changes it in this PR?

Comment (BigLep) [reply github/filecoin-project.yml:3466]: This is related to state being out of sync.  I've asked in the IPDX/FilOz channel the right way to clear this up: https://filecoinproject.slack.com/archives/C06CY12V83S/p1726674153573169

I'm also running the sync workflow to see if that clears things up: https://github.com/filecoin-project/github-mgmt/actions/runs/10925569479

We shouldn't merge until the terraform plan is only touching resources we'd expect in the way we'd expect

Comment (BigLep) [reply github/filecoin-project.yml:3466]: Once I get the state sync figured out (i.e., no unrelated changes) are you good with merging @jennijuju ?

**Pull request merged**